recipe/meta.yaml

```diff
@@ -57,6 +60,7 @@ requirements:
     - cuda-cudart-dev   # [(cuda_compiler_version or "").startswith("12")]
     - cuda-nvml-dev     # [(cuda_compiler_version or "").startswith("12")]
     - cuda-nvtx-dev     # [(cuda_compiler_version or "").startswith("12")]
+    - cuda-nvcc-tools  # [(cuda_compiler_version or "").startswith("12")]
```

Comment:
Is there a way to confirm that JAX is using `nvvm` from here? Does it have some config paths we can check in the test phase?

Reply:
I have no idea. 

@mwittgen would you know how to do this/where to look? We sadly don't have a GPU device in the CI to test it.

Reply:
Ah was hoping this kind of check would not actually use the GPU. Sort of like `python -m sysconfig` for JAX. Poking around it looks like they have [`jax.print_environment_info()`]( https://jax.readthedocs.io/en/latest/_autosummary/jax.print_environment_info.html ). Maybe we could try that?

If we are unable to figure out an appropriate config check, we could [configure CI to use a GPU]( https://github.com/Quansight/open-gpu-server?tab=readme-ov-file#gpu-runners ). Looks like some work has already been done here. So this would just be adding another runner type

Another option we have made heavy use of elsewhere is caching the built packages on CI and then downloading them for local testing ( https://github.com/conda-forge/conda-forge.github.io/issues/1424 ). This may be more useful in the end as it is easier to iterate

Reply:
> Poking around it looks like they have [jax.print_environment_info()](https://jax.readthedocs.io/en/latest/_autosummary/jax.print_environment_info.html). Maybe we could try that? 

I looked at its implementation and it doesn't provide anything useful for the case we are testing on. 

> If we are unable to figure out an appropriate config check, we could [configure CI to use a GPU](https://github.com/Quansight/open-gpu-server?tab=readme-ov-file#gpu-runners). Looks like some work has already been done here. So this would just be adding another runner type

This seems like rather expensive to add a GPU here to do a simple test. 

Overall, it seems that this will need input from someone where it doesn't work. The current PR is an improvement on the status quo and I don't want to hold up the release anymore, thus I'm going to merge.